### ISSUE TITLE
How are the training set and validation set divided for the AGAN dataset?

### ISSUE THREAD

**Issue by fh9892** [joshyZhou/AST#19]
Regarding the AGAN dataset, I noticed that your code uses 'test_a' as the validation set during training, and also uses 'test_a' as the test set during the testing phase. May I ask if you are using the same 'test_a' for both the validation set and the test set for the AGAN dataset?
